py/examples/file_upload.py

```diff
@@ -7,17 +7,21 @@
 
 @app('/demo')
 async def serve(q: Q):
-    if 'file_upload' in q.args:
+    if 'file_upload' in q.args or 'file_upload_compact' in q.args:
```

Comment:
Can we break this into two separate examples? The UI of the example looks odd.

Reply:
sure